test/unit/models/RequestFundReleaseRequestSpec.scala

```diff
@@ -153,4 +155,72 @@ class RequestFundReleaseRequestSpec extends PlaySpec {
 
   }
 
+  private def extractErrorValidation(result: JsResult[FundReleasePropertyDetails]) = {
+    result match {
+      case JsError(errors) => ErrorConverter.convert(errors)
+      case _ => throw new Exception("no error message found")
+    }
+  }
+
+  private def createJson(nameOrNumber: String, postalCode: String): JsObject =
+    Json.obj(
+      "nameOrNumber" -> nameOrNumber,
+      "postalCode" -> postalCode
+    )
+
+  "validate nameOrNumber" when {
+
+    "nameOrNumber is Empty" in {
+
+      val invalidNameOrNumberJson = createJson("", "AA11 1AA")
+
+      extractErrorValidation(invalidNameOrNumberJson.validate[FundReleasePropertyDetails]) must contain(ErrorValidation("INVALID_NAME_OR_NUMBER", "Enter nameOrNumber", Some("/nameOrNumber")))
+    }
+
+    "nameOrNumber is too long" in {
+
+      val invalidNameOrNumberJson = createJson("aaaaaaaaaabbbbbbbbbbccccccccccdddddd", "AA11 1AA")
```

Comment:
This is much easier to discern how many chars there are :)
```suggestion
      val invalidNameOrNumberJson = createJson("a" * 36 , "AA11 1AA")
```

Reply:
Yeah this makes more sense
changes done